wp-content/themes/goonj-crm/functions.php

```diff
@@ -191,7 +191,30 @@ function goonj_handle_user_identification_form() {
 		}
 
 		// If we are here, then it means the user exists as an inducted volunteer.
-		wp_redirect(get_home_url() . "/collection-camp-form/#?source_contact_id=" . $contact['id'] );
+		// Fetch the most recent collection camp activity based on the creation date
+		$collectionCampResult = civicrm_api3('Activity', 'get', [
```

Comment:
Switch to APIv4

Reply:
I'm checking on APIv3 where I was able to get the fields only by using the simple query  activity But as I was using the API4 I was not able to get those custom fields

```
$result = civicrm_api3('Activity', 'get', [
  'sequential' => 1,
  'activity_type_id' => "Collection Camp Intent",
  'status_id' => "Under Authorization",
]);
```

Output
```
{
            "id": "2433",
            "activity_type_id": "61",
            "activity_date_time": "2024-08-03 14:52:40",
            "status_id": "10",
            "priority_id": "2",
            "is_test": "0",
            "is_auto": "0",
            "is_current_revision": "1",
            "is_deleted": "0",
            "is_star": "0",
            "created_date": "2024-08-03 14:52:40",
            "modified_date": "2024-08-03 14:52:40",
            "custom_69": "fads",
            "custom_70": "1101",
            "custom_71": "4954",
            "custom_72": "fasdfadsf",
            "custom_73": "2024-08-03 14:52:00",
            "custom_74": "2024-08-14 14:52:00",
            "source_contact_id": "1913",
            "source_contact_name": "Mrs. Akshata Naik",
            "source_contact_sort_name": "Naik, Akshata"
        },
```
**API4**
Where when I was using this same on the APIV4 then it didn't give me the custom fields as it gave me on API3
```
$activities = \Civi\Api4\Activity::get(FALSE)
  ->addWhere('activity_type_id', '=', 61)
  ->addWhere('status_id', '=', 10)
  ->addWhere('source_contact_id', '=', 1913)
  ->addOrderBy('created_date', 'DESC')
  ->setLimit(25)
  ->execute();
foreach ($activities as $activity) {
  // do something
}
```

output
```
{
    "id": 2433,
    "source_record_id": null,
    "activity_type_id": 61,
    "subject": null,
    "activity_date_time": "2024-08-03 14:52:40",
    "duration": null,
    "location": null,
    "phone_id": null,
    "phone_number": null,
    "details": null,
    "status_id": 10,
    "priority_id": 2,
    "parent_id": null,
    "is_test": false,
    "medium_id": null,
    "is_auto": false,
    "relationship_id": null,
    "is_current_revision": true,
    "original_id": null,
    "result": null,
    "is_deleted": false,
    "campaign_id": null,
    "engagement_level": null,
    "weight": null,
    "is_star": false,
    "created_date": "2024-08-03 14:52:40",
    "modified_date": "2024-08-03 14:52:40"
  }
```
I'm checking how we get those fields also using the API4